app/controllers/games_controller.rb

```diff
@@ -1,2 +1,26 @@
 class GamesController < ApplicationController
+
+	def index
+		@games = Game.all
+		@open_games = Game.where(:black_uid => nil)
```

Comment:
This won't capture games that don't have a white_uid.  My guess is that you are going to always assign the first user to the white side of the board.  This works but removes the option for users to choose a color.  Just want to make sure you understand the tradeoff.